zig test is failing

```
+- run test
   +- zig test Debug native 7 errors
src/main.zig:7:10: error: no field named 'Fn' in enum '@typeInfo(builtin.Type).@"union".tag_type.?'
/opt/hostedtoolcache/zig/master/x64/lib/std/builtin.zig:259:18: note: enum declared here
src/main.zig:7:10: error: no field named 'Fn' in enum '@typeInfo(builtin.Type).@"union".tag_type.?'
/opt/hostedtoolcache/zig/master/x64/lib/std/builtin.zig:259:18: note: enum declared here
src/main.zig:7:10: error: no field named 'Fn' in enum '@typeInfo(builtin.Type).@"union".tag_type.?'
/opt/hostedtoolcache/zig/master/x64/lib/std/builtin.zig:259:18: note: enum declared here
src/main.zig:7:10: error: no field named 'Fn' in enum '@typeInfo(builtin.Type).@"union".tag_type.?'
/opt/hostedtoolcache/zig/master/x64/lib/std/builtin.zig:259:18: note: enum declared here
src/main.zig:7:10: error: no field named 'Fn' in enum '@typeInfo(builtin.Type).@"union".tag_type.?'
/opt/hostedtoolcache/zig/master/x64/lib/std/builtin.zig:259:18: note: enum declared here
src/main.zig:7:10: error: no field named 'Fn' in enum '@typeInfo(builtin.Type).@"union".tag_type.?'
/opt/hostedtoolcache/zig/master/x64/lib/std/builtin.zig:259:18: note: enum declared here
src/main.zig:7:10: error: no field named 'Fn' in enum '@typeInfo(builtin.Type).@"union".tag_type.?'
/opt/hostedtoolcache/zig/master/x64/lib/std/builtin.zig:259:18: note: enum declared here
error: the following command failed with 7 compilation errors:
/opt/hostedtoolcache/zig/master/x64/zig test -ODebug -Mroot=/home/runner/work/fiat-crypto/fiat-crypto/fiat-zig/src/main.zig --cache-dir /home/runner/work/fiat-crypto/fiat-crypto/fiat-zig/.zig-cache --global-cache-dir /home/runner/.cache/zig --name test --zig-lib-dir /opt/hostedtoolcache/zig/master/x64/lib/ --listen=- 
Build Summary: 0/3 steps succeeded; 1 failed
test transitive failure
+- run test transitive failure
   +- zig test Debug native 7 errors
error: the following build command failed with exit code 1:
/home/runner/work/fiat-crypto/fiat-crypto/fiat-zig/.zig-cache/o/477652a2b52bacc0f5683ede13f1acab/build /opt/hostedtoolcache/zig/master/x64/zig /opt/hostedtoolcache/zig/master/x64/lib /home/runner/work/fiat-crypto/fiat-crypto/fiat-zig /home/runner/work/fiat-crypto/fiat-crypto/fiat-zig/.zig-cache /home/runner/.cache/zig --seed 0xdda967b7 -Zf3eb92b2a3f10887 test
```

cc @jedisct1

--- a/.github/workflows/zig.yml
+++ b/.github/workflows/zig.yml
@@ -19,7 +19,7 @@ jobs:
       - name: Install Zig
         uses: goto-bus-stop/setup-zig@v2
         with:
-          version: master
+          version: 0.13.0
       - uses: actions/checkout@v4
       - name: Test Zig files
         run: (cd fiat-zig && zig build && zig build test)

